dash/extract-meta.js

```diff
@@ -20,13 +23,18 @@ const metadata = Object.create(null);
 componentPaths.forEach(componentPath =>
     collectMetadataRecursively(componentPath)
 );
-writeOut(metadata);
+if (failed) {
+    console.error('\nextract-meta failed.\n');
+}
+else {
+    writeOut(metadata);
+}
```

Comment:
we don't write a response if the error was severe enough for us to set `failed` - this causes `generate_components` to fail as well.